public/app/features/alerting/unified/components/Expression.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC } from 'react';
+import { Field, FieldSet, Input, FormAPI } from '@grafana/ui';
+
+type Props = FormAPI<any>;
+
+const Expression: FC<Props> = ({ register }) => {
+  return (
+    <FieldSet label="Create a query (expression) to be alerted on">
+      <Field>
+        <Input ref={register()} name="expression" placeholder="Enter a PromQL query here" />
```

Comment:
or Loki! There are "PromqlQueryField" and "LokiQueryField" components in grafana, I haven't tried those yet though